site/diagrams/sequenceDiagrams/command_control_concept_http.txt

```diff
@@ -0,0 +1,18 @@
+participant Device
+participant HTTP Adapter
+participant AMQP 1.0 Network
+participant Application
+
+Device->HTTP Adapter: 1. send event (hono-ttd=30)
+activate HTTP Adapter
+HTTP Adapter->Application: 2. connected notification (ttd=30)
```

Comment:
this is not a specific *notification* but simply the transfer of the event to the application, isn't it?

Reply:
It could be an event with notification or telemetry with notification.. but only the notification part is relevant for the following seqence.

Reply:
not sure what you mean by *notification*. FMPOV there is no such thing.